pkgs/tools/misc/ili2c/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, jdk, ant, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name = "ili2c-${version}";
+  version = "4.7.11";
+
+  buildInputs = [ ant jdk makeWrapper ];
+
+  src = fetchFromGitHub {
+    owner = "claeis";
+    repo = "ili2c";
```

Comment:
`repo = pname;` is a pretty common thing to do.

Reply:
Good idea. Changed with https://github.com/NixOS/nixpkgs/compare/b6388f7be50dbb30e56b18acb727fcabd0379fb5..2aaa8876ff68e454ff60fc9a2b6d2174df6bb17a#diff-efa52252ac746605798baf0ce1e25f13R11